common/src/main/java/no/difi/meldingsutveksling/config/IntegrasjonspunktPropertiesValidator.java

```diff
@@ -20,5 +20,12 @@ public void validate(Object target, Errors errors) {
             ValidationUtils.rejectIfEmpty(errors, "noarkSystem.endpointURL", "empty_field", "DPO enabled - cannot be null");
         }
 
+        if (props.getFeature().isEnableDPV()) {
+            ValidationUtils.rejectIfEmpty(errors, "altinnPTV.externalServiceCode", "empty_field", "DPV enabled - cannot be null");
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "DPV enabled - cannot be null" 4 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-app01.difi.local:9000/coding_rules#rule_key=squid%3AS1192)